layouts/shortcodes/grid1.html

```diff
@@ -121,12 +106,4 @@
 {{- else }}
   {{- $items = (index (.Inner | transform.Unmarshal) "item") }}
 {{- end }}
-{{- range $key, $d := $items -}}
-  {{- if eq $d.type "card" }}
-    {{- partial "_elements/card.html" $d -}}
-  {{- else }}
-    {{- partial "_elements/item.html" $d -}}
-  {{- end }}
-{{- end }}
-  </div>
-</div>
+{{- partial "_elements/grid.html" (merge .Params (dict "items" $items)) -}}
```

Comment:
Same question on `.Params` as with the other PR. May well be necessary, just checking.

Reply:
We have two optional parameters `outline` and `columns`.